Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/FastDepthAwareBlurHor.azsl

```diff
@@ -93,7 +93,7 @@ ShaderResourceGroup PassSrg : SRG_PerPass
         // The higher this value, the sharper edges will appear
         float m_depthFalloffStrength;
 
-        uint3 PADDING;
+        uint m_padding;
```

Comment:
same

Reply:
As said previously validation error messages come up with uint3 for padding. I will see if this is the case on later version of android with updated vulkan driver.

Reply:
Looks like with the device update the error message is no longer showing up with uint3 padding, i'll revert back.